getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -109,3 +121,10 @@ and ARMv8 is usually sufficient to cover most devices in use today.
 You can optimize the size further by compiling an Android export template with
 only the features you need. See :ref:`doc_optimizing_for_size` for more
 information.
+
+Further reading
+---------------
+
+- Article covering the Android export process [at GodotLearn.com](https://godotlearn.com/godot-3-1-how-to-export-for-android/) (*Uses Godot v3.1*).
```

Comment:
The link format is wrong, it should be of this form:
```
`at GodotLearn.com <https://godotlearn.com/godot-3-1-how-to-export-for-android/>`_
```

However, apart from additional screenshots and some more details on specific steps, this external documentation is mostly redundant with this page, and we have no control over it so it might well become outdated as Godot evolves (the export workflow for 3.2 already changed compared to 3.1.2).

So I'd prefer that we improve this page to make it exhaustive instead of referencing thirdparty content.
We do refer to thirdparty content on topics which are not Godot-specific, e.g. matrix calculus, etc., but for Godot's export workflow, there's no reason for the official documentation not to be the reference.

Reply:
That article has  more details on things beyond the minimum, so I think it would be good, but you and another maintainer would prefer no link so I'll just remove it.